internal/pkg/service/cli/cmd/remote/job/run.go

```diff
@@ -0,0 +1,112 @@
+package job
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/keboola/go-client/pkg/keboola"
+	"github.com/spf13/cobra"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/helpmsg"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/options"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+	"github.com/keboola/keboola-as-code/pkg/lib/operation/project/remote/job/run"
+)
+
+func RunCommand(p dependencies.Provider) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   `run [branch1/]component1/config1 [branch2/]component2/config2 ...`,
+		Short: helpmsg.Read(`remote/job/run/short`),
+		Long:  helpmsg.Read(`remote/job/run/long`),
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) (cmdErr error) {
+			// Ask for host and token if needed
+			baseDeps := p.BaseDependencies()
+			if err := baseDeps.Dialogs().AskHostAndToken(baseDeps); err != nil {
+				return err
+			}
+
+			// Get dependencies
+			d, err := p.DependenciesForRemoteCommand()
+			if err != nil {
+				return err
+			}
+
+			options, err := parseJobRunOptions(d.Options(), args)
+			if err != nil {
+				return err
+			}
```

Comment:
Drobnost, najpr to po mne chce token, az tak to parsuje/validuje argumenty:
![image](https://user-images.githubusercontent.com/19371734/218675574-09f062ac-124b-45c0-b08f-53c609106892.png)

Idealne by to malo byt opacne.